pol-core/bscript/compiler/astbuilder/SourceFileProcessor.cpp

```diff
@@ -0,0 +1,38 @@
+#include "SourceFileProcessor.h"
+
+#include "../clib/fileutil.h"
+
+#include "BuilderWorkspace.h"
```

Comment:
If you want to keep the "no file from the same folder unless .h/.cpp pair"
```suggestion
#include "compiler/astbuilder/BuilderWorkspace.h"
```

Reply:
I see I missed a few others